infrastructure/aws/ecs.tf

```diff
@@ -0,0 +1,132 @@
+data "aws_caller_identity" "current" {}
+
+resource "aws_kms_key" "ecs_cluster_key" {
+  description             = "example"
+  deletion_window_in_days = 7
+}
+
+resource "aws_cloudwatch_log_group" "audiozpravy_ecs_cluster" {
+  name = "audiozpravy-cluster"
+}
+
+resource "aws_ecs_cluster" "audiozpravy_ecs_cluster" {
+  name = "audiozpravy-cluster"
+
+  configuration {
+    execute_command_configuration {
+      kms_key_id = aws_kms_key.ecs_cluster_key.arn
+      logging    = "OVERRIDE"
+
+      log_configuration {
+        cloud_watch_encryption_enabled = true
+        cloud_watch_log_group_name     = aws_cloudwatch_log_group.audiozpravy_ecs_cluster.name
+      }
+    }
+  }
+}
+
+
+
+resource "aws_ecs_task_definition" "backend" {
+  family = "backend"
+  network_mode = "awsvpc"
+  requires_compatibilities = [ "FARGATE" ]
+  execution_role_arn = aws_iam_role.ecs_backend_task_execution_role.arn
+  task_role_arn = aws_iam_role.ecs_backend_task_role.arn
+  cpu       = "256"
+  memory    = "1024"
```

Comment:
Minimalni validni FARGATE configurace je 256 CPU a 512 Memory, zvladne to bezet v tom,  at setrime.

Reply:
Fixed